### PR TITLE
gexiv2 0.10.3

### PR DESCRIPTION
**Pull request by alistairmcmillan**

### CODE REVIEW

**Comment by tdsmith** [comment]
Have you seen / do you have any comment on https://github.com/Homebrew/homebrew/pull/38428?

**Comment by alistairmcmillan** [comment]
Hi, this is updated to include bors-ltd's changes. It passes all tests. Please let me know if you need me to do anything else.

File: Library/Formula/gexiv2.rb [hunk 1]
```diff
@@ -14,11 +14,14 @@ class Gexiv2 < Formula
   depends_on "pkg-config" => :build
   depends_on "glib"
   depends_on "exiv2"
+  depends_on "gobject-introspection"
```
Comment (tdsmith) [inline Library/Formula/gexiv2.rb]: Can you make this a build dependency?

(Also, nit: can you sort the :build dependencies first?)

**Comment by alistairmcmillan** [comment]
Updated as requested. Let me know if I should make any other changes.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**